### PR TITLE
This is my first commit

### PR DESCRIPTION
**Pull request by LibertyToad**

### CODE REVIEW

**Review by ScotMeyer — COMMENTED** [review]
Why oh why did you do this?  It looks great!

File: _posts/0000-01-03-LibertyToad.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Why is my avatar is angry?
```
Comment (ScotMeyer) [inline _posts/0000-01-03-LibertyToad.md:6]: If you flip the avatar upside down then it's happy.

**Review by whatupfoo — APPROVED** [review]

**Review by ScotMeyer — APPROVED** [review]

**Comment by ScotMeyer** [comment]
Nice job

File: _posts/0000-01-03-LibertyToad.md (hunk 1 above)
Comment (LibertyToad) [reply _posts/0000-01-03-LibertyToad.md:6]: HA

**Pull request merged**